src/ziggurat/metrics.clj

```diff
@@ -9,13 +9,8 @@
            [io.dropwizard.metrics5 Histogram Meter MetricName MetricRegistry]
            java.util.concurrent.TimeUnit)
   (:gen-class
-   :name tech.gojek.ziggurat.internal.Metrics
-   :methods [^{:static true} [incrementCount [String String] void]
-             ^{:static true} [incrementCount [String String java.util.Map] void]
-             ^{:static true} [decrementCount [String String] void]
-             ^{:static true} [decrementCount [String String java.util.Map] void]
-             ^{:static true} [reportTime [String long] void]
-             ^{:static true} [reportTime [String long java.util.Map] void]]))
+   :methods [^{:static true} [incrementCount [String String] void] ^{:static true} [incrementCount [String String java.util.Map] void] ^{:static true} [decrementCount [String String] void] ^{:static true} [decrementCount [String String java.util.Map] void] ^{:static true} [reportTime [String long] void] ^{:static true} [reportTime [String long java.util.Map] void]]
```

Comment:
is there a reason that everything is moved into the same line? This makes it hard to read

Reply:
reverted...